storage/system-test/encryption.test.js

```diff
@@ -54,24 +54,38 @@ describe('storage:encryption', function () {
 
   describe('uploadEncryptedFile', function () {
     it('should upload a file', function (done) {
-      program.uploadEncryptedFile(bucketName, filePath, fileName, key, function (err, file) {
+      var options = {
+        bucket: bucketName,
+        srcFile: filePath,
+        destFile: fileName,
+        key: key
+      };
+
+      program.uploadEncryptedFile(options, function (err, file) {
```

Comment:
All storage samples which before took more than one positional argument (besides the callback) have now collapsed them into a single `options` argument. This cleans up the function signature while also making it easier for the CLI to call the sample methods e.g. [`uploadEncryptedFile(options, callback)`](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/storage/storage/encryption.js#L58)

The JSDoc comment above each sample method very clearly documents the `options` argument and its properties.